build/geanyvc.m4

```diff
@@ -6,13 +6,12 @@ AC_DEFUN([GP_CHECK_GEANYVC],
         AC_HELP_STRING([--enable-gtkspell=ARG],
             [Enable GtkSpell support in GeanyVC. [[default=auto]]]),,
         enable_gtkspell=auto)
-    GP_CHECK_GTK3([gtkspell_package=gtkspell3-3.0],
-                  [gtkspell_package=gtkspell-2.0])
+    GTKSPELL_PACKAGE_NAME=gtkspell3-3.0
```

Comment:
Could you reuse the old name to simplify the diff?  I don't see much reason to change the variable name, is there any?